src/advanced-bloom/AdvancedBloomFilter.ts

```diff
@@ -158,42 +208,68 @@ export class AdvancedBloomFilter extends Filter
     get threshold(): number { return this._extractFilter.threshold; }
     set threshold(value: number) { this._extractFilter.threshold = value; }
 
-    /** Sets the kernels of the Blur Filter */
+    /** The kernels of the Blur Filter */
     get kernels(): number[] { return this._blurFilter.kernels; }
     set kernels(value: number[]) { this._blurFilter.kernels = value; }
 
     /**
-     * Sets the strength of the Blur properties simultaneously
+     * The strength of the Blur properties simultaneously
      * @default 2
      */
     get blur(): number { return this._blurFilter.strength; }
     set blur(value: number) { this._blurFilter.strength = value; }
 
     /**
-     * Sets the quality of the Blur Filter
+     * The quality of the Blur Filter
      * @default 4
      */
     get quality(): number { return this._blurFilter.quality; }
     set quality(value: number) { this._blurFilter.quality = value; }
 
     /**
-     * Sets the pixelSize of the Kawase Blur filter
+     * The pixel size of the Kawase Blur filter
      * @default {x:1,y:1}
      */
     get pixelSize(): PointData { return this._blurFilter.pixelSize; }
-    set pixelSize(value: PointData) { this._blurFilter.pixelSize = value; }
+    set pixelSize(value: PointData | DeprecatedPixelSizeValue)
+    {
+        if (typeof value === 'number' || Array.isArray(value))
+        {
+            deprecation(v8_0_0, 'AdvancedBloomFilter.pixelSize now only accepts {x, y} PointData type.');
+            this._blurFilter.pixelSize = convertDeprecatedPixelSize(value);
+
+            return;
+        }
+
+        this._blurFilter.pixelSize = value;
+    }
 
     /**
-     * Sets the pixelSize of the Kawase Blur filter on the `x` axis
+     * The horizontal pixelSize of the Kawase Blur filter
      * @default 1
      */
     get pixelSizeX(): number { return this._blurFilter.pixelSizeX; }
     set pixelSizeX(value: number) { this._blurFilter.pixelSizeX = value; }
 
     /**
-     * Sets the pixelSize of the Kawase Blur filter on the `y` axis
+     * The vertical pixel size of the Kawase Blur filter
      * @default 1
      */
     get pixelSizeY(): number { return this._blurFilter.pixelSizeY; }
     set pixelSizeY(value: number) { this._blurFilter.pixelSizeY = value; }
 }
+
+function convertDeprecatedPixelSize(value: DeprecatedPixelSizeValue): PointData
```

Comment:
I would inline this function in the pixelSize setter since you don't need to use it in the constructor.